listenbrainz/db/msid_mbid_mapping.py

```diff
@@ -144,18 +112,38 @@ def _update_items_from_map(models: Dict[str, List[ModelT]], metadatas: Dict, rem
                     # in messybrainz either. note that we always submit the msid from website if available so this
                     # case shouldn't occur unless some now playing case is mishandled or someone else uses the api
                     # to submit say a pinned recording with a mbid for which LB has never seen a listen.
-                    current_app.logger.info("Couldn't find data for item in mapping and item doesn't have msid: %s", item)
+                    current_app.logger.info("Couldn't find data for item in mapping and item doesn't have msid: %s",
```

Comment:
This as an info is a bit odd. Should it be an error/warn or be removed?

Reply:
We had it as an error and there was too many errors (mostly from same users) so we decided to degrade it to info for the time being.

Reply:
https://github.com/metabrainz/listenbrainz-server/pull/2098

Reply:
Thinking more, I agree it makes sense to upgrade to `warn` now.